step-framework-parent/step-framework-collections/src/main/java/step/core/accessors/CachedAccessor.java

```diff
@@ -85,7 +85,12 @@ public T findByCriteria(Map<String, String> criteria) {
 		return cache.findByCriteria(criteria);
 	}
 
-	@Override
+    @Override

```

Comment:
Formatting indentation issue
Use directly a List, no need of collection and use "import" rather than having the package name here